### ISSUE TITLE
Occasionally missing Accessibility Label

### ISSUE THREAD

**Issue by h0shy** [SRGSSR/srgletterbox-apple#268]
### Issue type

Incorrect behavior

### Description of the problem

In Voice Over, the a11y label of the entire Video, which reads "Video, zweimal tippen um Bedienelemente anzuzeigen." doesn't always get read out. In random cases, it just reads "Video".

### Environment information

> The simulator is a valid device as well. If all versions or devices seem to be affected, simply enter 'Any'

* Library version: 8.0.0
* iOS 15.5
* Device: iPhone 13 Pro Max

### Reproducibility

Not always reproducible.

### Steps to reproduce

Focus the Video in Voice Over.

**Comment by defagos** [comment]
Hi. 

Thanks for your report and sorry for the late reply. I'll have a look, as this issue can quite easily be reproduced with VoiceOver enabled:

1. Focus on the video. The accessibility hint is provided (double tap action).
2. Double tap to toggle the controls. Now the accessibility hint is missing.

**Comment by defagos** [comment]
When describing the player view for VoiceOver we are using two pieces of text:

1. A short one, the [accessibility label](https://developer.apple.com/documentation/objectivec/nsobject/1615181-accessibilitylabel), that describes the element.
2. A longer one which describes what the associated action does, the [accessibility hint](https://developer.apple.com/documentation/objectivec/nsobject/1615093-accessibilityhint).

I suspect that an accessibility hint is only spoken the first time the control it is associated with is focused. If you namely use VoiceOver with a switch button (e.g. in the system Settings app), you can hear that the hint (usually "Double tap to toggle setting") is only spoken the first time, not after you toggle the switch without leaving the control. This is likely implemented this way to avoid unnecessary verbosity and matches what we implemented for our player view (whose controls are toggled like a switch would be).

To have the full label always spoken we would need to include the action in the accessibility label itself, which is not what [Apple recommends](https://developer.apple.com/library/archive/documentation/UserExperience/Conceptual/iPhoneAccessibility/Making_Application_Accessible/Making_Application_Accessible.html), as labels must be short. So the situation we have, which avoids being verbose about a hint while the focus is kept on a control, seems correct to me.

Of course if you can reproduce a use case where the hint is missing while moving the control to the player view, then we might still have another bug.
